### PR TITLE
update hermes precheck conditional

### PR DESCRIPTION
**Pull request by bderusha**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/hermes/tasks/main.yml [hunk 1]
```diff
@@ -168,7 +168,7 @@
     owner: "{{ hermes_user }}"
     group: "{{ hermes_user }}"
     mode: "0750"
-  when: ansible_system_vendor == "Amazon EC2"
+  when: ansible_kernel == "4.4.0-1079-aws"
```
Comment (jdmulloy) [inline playbooks/roles/hermes/tasks/main.yml]: This is going to change the next time the AMI is updated.

**Review by jdmulloy — CHANGES_REQUESTED** [review]

**Comment by bderusha** [comment]
@jdmulloy ready for another pass

**Review by coryleeio — APPROVED** [review]

**Comment by jdmulloy** [comment]
@bderusha I chatted with @coryleeio and we decided that a simple switch would be simpler and more reliable. See #5219

**Comment by bderusha** [comment]
:+1:

**Pull request closed without merging**